finetune/finetune_power.py

```diff
@@ -173,25 +190,19 @@ def main(args: argparse.Namespace) -> None:
         cfg.PG.VAL.BATCH_SIZE,
         False,
     )
-    test_dataloader = create_dataloader(
-        cfg.PG.TEST.START_TIME,
-        cfg.PG.TEST.END_TIME,
-        cfg.PG.TEST.FREQUENCY,
-        cfg.PG.TEST.BATCH_SIZE,
-        False,
-    )
 
-    model = setup_model("PanguPowerConvSigmoid", device)
+    model = setup_model("PanguPowerConv", device)
```

Comment:
Shouldn't the model type be passed as an argument to the function or saved in the configuration?

Reply:
Good point! I'll put this in the config